src/App.js

```diff
@@ -1,5 +1,51 @@
+import { Console } from "@woowacourse/mission-utils";
+import RaceController from "./controllers/RaceController.js";
+import Validator from "./utils/Validator.js";
+import UserInput from "./views/UserInput.js";
+import DisplayController from "./views/DisplayController.js";
+
 class App {
-  async run() {}
+  constructor() {
+    this.userInput = new UserInput();
+    this.validator = new Validator();
+    this.raceController = new RaceController();
+    this.displayController = new DisplayController();
+  }
+
+  async run() {
+    // Get user input
+    const userInput = await this.userInput.getUserInputCars();
+
+    // Split user input by separator
+    const userInputArray = userInput.split(",");
```

Comment:
```suggestion
    // Get user input
    const userInputCar = await this.userInput.getUserInputCars();

    // Split user input by separator
    const userInputCars = userInput.split(",");
```
아래에서는 `userInputCount`로 어떤 변수인지 명확하게 나타내주셔서 여기도 자동차 관련된 변수라는걸 명시해도 좋을 것 같아요!

Reply:
this.userInput 뒤에 있는 메서드 명으로 추측이 가능하지만 
wo-o29 님 말씀처럼 어떤 변수인지 명확하게 나타나면 이게 무슨 변수인지 한 번에 알 수 있어서 더 좋을 것 같아요!👍